packages/ember-views/lib/mixins/view_support.js

```diff
@@ -610,6 +613,7 @@ export default Mixin.create({
     this.scheduledRevalidation = false;
```

Comment:
yes, as in ES6 you can't touch `this` before calling super, we should slowly move our internals to this.